stable-25-1-3: ydb/tests/compatibility/test_followers.py.TestSecondaryIndexFollowers.test_secondary_index_followers[rolling_stable-25-1-2_to_stable-25-1-3-True]

https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/16550747006/ya-main-x86-64-asan/try_1/ya-test.html#FAIL

https://github.com/ydb-platform/ydb/actions/runs/16550747006

Падает тест(ы) совместимости 25-1-2 + 25-1-3 и 25-1-3 + main
ydb/tests/compatibility test_followers.py.TestSecondaryIndexFollowers.test_secondary_index_followers[rolling_stable-**25-1-2_to_stable-25-1-3**-True]

```
ydb/tests/compatibility/test_followers.py:227: in test_secondary_index_followers
    self.check_statistics(enable_followers)
ydb/tests/compatibility/test_followers.py:218: in check_statistics
    assert False, f"Expected reads but there is timeout waiting for read stats from '/Root/{self.TABLE_NAME}/{self.INDEX_NAME}/indexImplTable'"
E   AssertionError: Expected reads but there is timeout waiting for read stats from '/Root/table/idx/indexImplTable'
E   assert False

```

ydb/tests/compatibility test_followers.py.TestSecondaryIndexFollowers.test_secondary_index_followers[rolling_stable-**25-1-3_to_main**-True]
```
ydb/tests/compatibility/test_followers.py:224: in test_secondary_index_followers
    for _ in self.roll():
ydb/tests/library/compatibility/fixtures.py:266: in roll
    node.stop()
ydb/tests/library/harness/kikimr_runner.py:262: in stop
    super(KiKiMRNode, self).stop()
ydb/tests/library/harness/daemon.py:194: in stop
    if not self.__check_can_launch_stop("stop"):
ydb/tests/library/harness/daemon.py:171: in __check_can_launch_stop
    raise DaemonError(
E   ydb.tests.library.harness.daemon.DaemonError: Daemon failed with message: Unexpectedly finished before stop.
E   Process exit_code = -6.
E   Stdout file name: 
E   /home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_followers.py.TestSecondaryIndexFollowers.test_secondary_index_followers.rolling_stable-25-1-3_to_main-True/cluster/node_9/stdout
E   Stderr file name: 
E   /home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_followers.py.TestSecondaryIndexFollowers.test_secondary_index_followers.rolling_stable-25-1-3_to_main-True/cluster/node_9/stderr
E   Stderr content:
E   
E   GRpc memory quota was set but disabled due to issues with grpc quoter, to enable it use EnableGRpcMemoryQuota option
E   Current KQP shutdown state: spent 0 seconds, not started yet
E   Drain completed.
E   GRpc memory quota was set but disabled due to issues with grpc quoter, to enable it use EnableGRpcMemoryQuota option
E   ======= terminate() call stack ========
E   0. /tmp//-S/ydb/core/driver_lib/run/main.cpp:180: KikimrTerminateHandler @ 0x1334A132
E   1. /tmp//-S/contrib/libs/cxxsupp/libcxxrt/exception.cc:0: terminate @ 0x99F99C8
E   2. /tmp//-S/contrib/libs/cxxsupp/libcxxrt/exception.cc:821: report_failure @ 0x99FA4FD
E   3. /tmp//-S/contrib/libs/cxxsupp/libcxxrt/exception.cc:0: __cxa_rethrow @ 0x99FA4FD
E   4. /tmp//-S/ydb/library/actors/core/actor.cpp:410: Receive @ 0xA63AC38
E   5. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:268: Execute @ 0xA66AEF7
E   6. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:458: operator() @ 0xA66E9A2
E   7. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:510: ProcessExecutorPool @ 0xA66E570
E   8. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:536: ThreadProc @ 0xA66F192
E   9. /-S/util/system/thread.cpp:245: ThreadProxy @ 0x9A937F7
E   10. ??:0: ?? @ 0x7F3D00466AC2
E   11. ??:0: ?? @ 0x7F3D004F884F
E   ======== exception call stack =========
E   0. /tmp//-S/contrib/libs/cxxsupp/libcxxrt/exception.cc:839: throw_exception @ 0x99F9DD1
E   1. /tmp//-S/contrib/libs/cxxsupp/libcxxrt/exception.cc:882: __cxa_throw @ 0x99F9D76
E   2. /-S/util/generic/yexception.cpp:168: ThrowYException @ 0x9A5DA80
E   3. /tmp//-S/ydb/core/tablet/tablet_sys.cpp:647: HandleFollowerDisconnect @ 0x17530E6F
E   4. /tmp//-S/ydb/core/tablet/tablet_sys.cpp:2117: InterconnectSessionDisconnected @ 0x17530E6F
E   5. /tmp//-S/ydb/library/actors/core/actor.cpp:406: Receive @ 0xA63AB8B
E   6. /tmp//-S/contrib/libs/cxxsupp/libcxxrt/exception.cc:1008: __cxa_rethrow @ 0x99FA473
E   7. /tmp//-S/ydb/library/actors/core/actor.cpp:410: Receive @ 0xA63AC38
E   8. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:268: Execute @ 0xA66AEF7
E   9. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:458: operator() @ 0xA66E9A2
E   10. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:510: ProcessExecutorPool @ 0xA66E570
E   11. /tmp//-S/ydb/library/actors/core/executor_thread.cpp:536: ThreadProc @ 0xA66F192
E   12. /-S/util/system/thread.cpp:245: ThreadProxy @ 0x9A937F7
E   13. ??:0: ?? @ 0x7F3D00466AC2
E   14. ??:0: ?? @ 0x7F3D004F884F
E   =======================================
E   Terminating due to uncaught exception 0x3238694d8990    what() -> "ydb/core/tablet/tablet_sys.cpp:647: Condition violated: `it != LeaderInfo.end()'"
E   of type yexception

```

<!-- nemesis:related-versions:start -->

---

### Related issues

- Also filed (stable-26-3-1): #48643

<!-- nemesis:related-versions:end -->

--- a/ydb/tests/compatibility/test_followers.py
+++ b/ydb/tests/compatibility/test_followers.py
@@ -80,7 +80,7 @@ class TestFollowersCompatibility(MixedClusterFixture):
                       WITH (
                         AUTO_PARTITIONING_BY_SIZE = ENABLED,
                         AUTO_PARTITIONING_PARTITION_SIZE_MB = 1,
-                        READ_REPLICAS_SETTINGS = \"PER_AZ:1\"
+                        READ_REPLICAS_SETTINGS = \"ANY_AZ:1\"
                       );"""
                 )
                 id_ = 0
@@ -165,7 +165,7 @@ class TestSecondaryIndexFollowers(RollingUpgradeAndDowngradeFixture):
 
             if enable_followers:
                 alter_index_query = f"""
-                    ALTER TABLE `{self.TABLE_NAME}` ALTER INDEX `{self.INDEX_NAME}` SET READ_REPLICAS_SETTINGS "PER_AZ:1";
+                    ALTER TABLE `{self.TABLE_NAME}` ALTER INDEX `{self.INDEX_NAME}` SET READ_REPLICAS_SETTINGS "ANY_AZ:1";
                 """
                 session_pool.execute_with_retries(alter_index_query)
 

